src/Avalonia.Themes.Default/TextBox.xaml

```diff
@@ -3,6 +3,7 @@
     <Setter Property="Background" Value="{DynamicResource ThemeBackgroundBrush}"/>
     <Setter Property="BorderBrush" Value="{DynamicResource ThemeBorderMidBrush}"/>
     <Setter Property="BorderThickness" Value="{DynamicResource ThemeBorderThickness}"/>
+    <Setter Property="SelectionBrush" Value="{DynamicResource HighlightBrush}"/>
```

Comment:
The default for SelectionForegroundBrush can be defined here as well.

Reply:
Yes I think so and it needs to define the default brush in default theme may be as "HighlightForegroundBrush"